.github/workflows/linter.yml

```diff
@@ -37,14 +37,16 @@ jobs:
           #   - man/.* - all snippets in man pages (false positives due to
           #              missing shebangs)
           #   - .*\.(in|SKELETON) - all template/skeleton files
+          #                         except kernel-install
           #   - tools/coverity\.sh - external file (with some modifications)
-          FILTER_REGEX_EXCLUDE: .*/(man/.*|.*\.(in|SKELETON)|tools/coverity\.sh)$
```

Comment:
Oh well, this is unfortunate, but it is what it is, thanks for figuring this out.